cobrautil.go

```diff
@@ -173,24 +173,44 @@ func OpenTelemetryRunE(flagPrefix string, prerunLevel zerolog.Level) CobraRunFun
 
 		provider := strings.ToLower(MustGetString(cmd, prefixed("provider")))
 		serviceName := MustGetString(cmd, prefixed("service-name"))
-
+		endpoint, _ := cmd.Flags().GetString(prefixed("endpoint"))
+		
+		var exporter trace.SpanExporter
+		var err error
+
+		// If endpoint is not set, the clients are configured via the OpenTelemetry environment variables or
+		// default values.
```

Comment:
This message would probably be more useful in the flag description.

Reply:
Added an abbreviation in https://github.com/jzelinskie/cobrautil/pull/4/commits/e4781feb416e12a0dedc042095fadaaba6d83614